src/services/dataSrvc.js

```diff
@@ -38,5 +38,16 @@ angular.module('app').factory('DataSrvc', ['$http', '$q', '$location', function
           })
 
       },
+      getUrlData: function (url, cb) {
```

Comment:
Should be changed to method getPackageData, which returns data by package name, and repository URL as a constant in this class.